app/views/govuk_publishing_components/components/_intervention.html.erb

```diff
@@ -16,14 +16,31 @@
 
   intervention_helper = GovukPublishingComponents::Presenters::InterventionHelper.new(local_assigns)
   dismiss_href = intervention_helper.dismiss_link
+
+  suggestion_tag_options = {
+    class: "govuk-link gem-c-intervention__suggestion-link",
+    href: suggestion_link_url,
+    data: suggestion_data_attributes,
+  }
+
+  new_tab ||= false
+  if new_tab && (suggestion_link_text && suggestion_link_text)
+    target = "_blank"
+    rel = "noopener noreferrer"
+    rel << " external" unless suggestion_link_url && suggestion_link_url.start_with?("/", "https://gov.uk", "https://www.gov.uk")
+
+    suggestion_tag_options.merge!({ target: target, rel: rel })
+
+    accessible_text = " (opens in a new tab)"
+    suggestion_link_text << accessible_text unless suggestion_link_text.end_with?(accessible_text)
+  end
 %>
-<% if suggestion_text or (suggestion_link_text and suggestion_link_text) %>
+<% if suggestion_text || (suggestion_link_text && suggestion_link_text) %>
```

Comment:
This isn't your change, but the part of this statement after `&&` looks redundant - isn't it the same as the below?

`<% if suggestion_text || suggestion_link_text %>`

Reply:
I don't think so because the banner can have just a link or just some text, whereas the check below is in case of both.

Reply:
I don't understand... the logic in the brackets after the `||` is and-ing two variables that are the same variable - `suggestion_link_text`. Can't the second `suggestion_link_text` be removed? Or am I going mad? How does this work?

Reply:
omg sorry I think that's a typo from before as it should be link_text and link_url